docs/offroadvoertuigen.md

```diff
@@ -0,0 +1,62 @@
+# Offroad Voertuigen lijst
+
+De Lijst hieronder geeft aan welke voertuigen vanuit de gemeente zijn toegestaan om te gebruiken buiten de geasfalteerde wegen, we klassifiseren dit onder:
+| Classificering | Uitleg | Omschrijving |
+|-|---|---|
+|V| Verharde Baan    | een verhard weg waarbij het wegdek uit asfalt of beton bestaat |

```

Comment:
|V| Verharde weg | een verharde weg waarbij het wegdek uit asfalt of beton bestaat |